### ISSUE TITLE
logitechmediaserver actually needs lock-in of certain CPAM modules.

### ISSUE THREAD

**Issue by poetnerd** [titou386/iocage-plugin-logitechmediaserver#4]
It is a sad reality that LMS is sensitive to the version of Perl that gets installed.

The folks there are well aware of this, and have locked in certain CPAM modules after various Perl updates result in LMS simply stopping working.

In my digging into this issue I have found two different approaches to solving this problem:

1. The regular distribution that point people at https://github.com/Logitech/slimserver-vendor that contains a complete set of CPAM libraries that continue to work on very old legacy systems.
2. An updated fork of that tree done by simone (whose instructions you refer to as the basis for your work.): https://github.com/simonefil/slimserver-vendor 

The current distribution of iocage-plugin-logitechmediaserver needs to use locked-in versions of CPAM modules, at least from simone's fork to be robust against breakage if Perl gets updated.

**Comment by titou386** [comment]
Thank you for the detailed explanation.
For this plugin, I'm using the original `slimserver-vendor` from Logitech. ( My `README` is misleading )
Is that what you wanted to know ?

**Comment by poetnerd** [comment]
Yes. Thank you.  It was difficult to determine if your build had done the version specific lock-in.

Reading the code more carefully, I have a follow-on question:

In post_install.sh, I see reference to "${perl_version}".  Does this mean that post_install.sh runs with the system-installed Perl version, and then pulls the CPAN snapshot specific to that version?
